integration_tests/test_dbapi.py

```diff
@@ -120,6 +119,7 @@ def test_select_tpch_1000(presto_connection):
 def test_cancel_query(presto_connection):
     cur = presto_connection.cursor()
     cur.execute("select * from tpch.sf1.customer")
+    cur.fetchone()
```

Comment:
how is this related?

Reply:
Sorry, I forgot to submit issue. It comes from  dispatcher . https://github.com/prestosql/presto/issues/2683

Reply:
let's add 

```
# TODO (https://github.com/prestosql/presto/issues/2683) test with and without .fetchone
```